test/suite.toml

```diff
@@ -0,0 +1,1357 @@
+["test/Mesh/BrickMesh.jl"]
```

Comment:
I would move the filename to it's own field and name the test `BrickMesh`

Reply:
I thought about that, but then we can have name clashes (many tests are called `runtests.jl`), and we'll need to include the full filename anyway.

Reply:
Spoke with Simon about this, we can use `basename(dirname(file))` when `basename(file) == runtests.jl`, so long this doesn't result in name clashes.

Reply:
Ugh, already hitting an issue with this: `advection_sphere_ssp33` is tested in `DGmethods` and `DGmethods_old`. Just adding `_old` to the header

Reply:
If someone finished out #491 this issue would go away.

Reply:
Yep, @trontrytel and I have been working on the microphysics drivers